src/sagemaker/workflow/pipeline.py

```diff
@@ -1043,7 +1039,7 @@ def get_function_step_result(
     if job_status == "Completed":
         return deserialize_obj_from_s3(
             sagemaker_session=sagemaker_session,
-            s3_uri=s3_path_join(s3_output_path, execution_id, step_name, RESULTS_FOLDER),
+            s3_uri=s3_path_join(s3_output_path),
```

Comment:
If I try to load results from an old execution, I guess it won’t find it because the output_config is different now in new executions.

Reply:
Will update the logic here to make it fall back to the old path: `s3_uri=s3_path_join(s3_output_path, execution_id, step_name, RESULTS_FOLDER),` if s3_output_path is incomplete